test/test-cases/functional/ptf/p4_dash_utils.py

```diff
@@ -0,0 +1,115 @@
+import grpc
```

Comment:
better add 1 example to show how to use it.

Reply:
Demo the use of `set_internal_config` in `saidashvnet_sanity.py`.